packages/EasyBatch/src/Serializer/Serializer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyBatch\Serializer;
```

Comment:
Why not `Serializers` (plural form)?

Reply:
We use the singular form in all packages, except `EasyActivity`.
@natepage - what do you think?

Reply:
But it EasyBatch all namespaces are in plural form.

Reply:
I see, thanks.
Dane.